src/api/firebase.js

```diff
@@ -50,6 +58,20 @@ export function getItemData(snapshot) {
 		 */
 		data.id = docRef.id;
 
+		let isDefaultChecked;
+
+		if (!data.dateLastPurchased) {
+			isDefaultChecked = false;
+		} else {
+			const dayInMilliseconds = 24 * 60 * 60 * 1000;
+			const date = data.dateLastPurchased.toDate();
+			const currentDate = new Date();
+			const timeElapsed = currentDate - date;
+			isDefaultChecked = timeElapsed < dayInMilliseconds;
```

Comment:
This looks really clean - nicely done!

Reply:
Totally agree with @emilysellers! Nicely done. One suggestion for improvement - since some of these day/time variables will probably show up in the future in other places, this would be a good opportunity to move them to a `utils.js` file and export them as constants (e.g. `export const DAYINMILLISECONDS = ...`. That way they can be reusable and not redefined every time you need them.